tests/db/test_sandboxMetadataDB.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import pytest
+
+from diracx.db.sandbox_metadata.db import SandboxMetadataDB
+
+
+@pytest.fixture
+async def sandbox_metadata_db(tmp_path):
+    sandbox_metadata_db = SandboxMetadataDB("sqlite+aiosqlite:///:memory:")
+    async with sandbox_metadata_db.engine_context():
+        yield sandbox_metadata_db
+
+
+async def test__get_put_owner(sandbox_metadata_db):
+    async with sandbox_metadata_db as sandbox_metadata_db:
+        result = await sandbox_metadata_db._get_put_owner("owner", "owner_group")
+        assert result == 1
+        result = await sandbox_metadata_db._get_put_owner("owner_2", "owner_group")
+        assert result == 2
+        result = await sandbox_metadata_db._get_put_owner("owner", "owner_group")
+        assert result == 1
+        result = await sandbox_metadata_db._get_put_owner("owner_2", "owner_group")
+        assert result == 2
+        result = await sandbox_metadata_db._get_put_owner("owner_2", "owner_group_2")
+        assert result == 3
+
+
+async def test_insert(sandbox_metadata_db):
+    async with sandbox_metadata_db as sandbox_metadata_db:
+        result = await sandbox_metadata_db.insert(
+            "owner",
+            "owner_group",
+            "sbSE",
+            "sbPFN",
+            123,
+        )
+        assert result == (1, True)
+
+        result = await sandbox_metadata_db.insert(
+            "owner",
+            "owner_group",
+            "sbSE",
+            "sbPFN",
+            123,
+        )
+        assert result == (1, False)
+
+        result = await sandbox_metadata_db.insert(
+            "owner_2",
+            "owner_group",
+            "sbSE",
+            "sbPFN_2",
+            123,
+        )
+        assert result == (2, True)
+
+        result = await sandbox_metadata_db.insert(
+            "owner",
+            "owner_group",
+            "sbSE",
+            "sbPFN_2",
+            123,
+        )
+        assert result == (0, False)
+
+
+async def test_delete(sandbox_metadata_db):
+    async with sandbox_metadata_db as sandbox_metadata_db:
+        result = await sandbox_metadata_db.insert(
+            "owner",
+            "owner_group",
+            "sbSE",
+            "sbPFN",
+            123,
+        )
+        assert result == (1, True)
+
+        result = await sandbox_metadata_db.delete(1)
```

Comment:
According to the type hint this should be:
```suggestion
        result = await sandbox_metadata_db.delete([1])
```